hook.py

```diff
@@ -1,17 +1,24 @@
 from plugins.builder.app.build_svc import BuildService
 from app.utility.base_world import BaseWorld
+from plugins.builder.app.builder_gui import BuilderGUI
 
 name = 'builder'
 description = 'Dynamically compile ability code via docker containers'
-address = None
+address = '/plugin/builder/gui'
 access = BaseWorld.Access.RED
 
 
 async def enable(services):
+    environments = BaseWorld.strip_yml('plugins/builder/conf/environments.yml')
     BaseWorld.apply_config('build', BaseWorld.strip_yml('plugins/builder/conf/environments.yml')[0])
     build_svc = BuildService(services)
     await build_svc.stage_enabled_dockers()
 
+    envs = environments[0]['enabled']
```

Comment:
This assumes that there's always an `'enabled'` object in the `environments.yml` file, should I add in a check for this and make it an empty list if `'enabled'` doesn't exist?

Reply:
Oh yeah, probably!